guides/Overmap Ships and Places - What's Needed.md

```diff
@@ -0,0 +1,59 @@
+Overmap Ships + Places, the Guide
```

Comment:
Why isn't this info in the overmap readme?

Reply:
https://github.com/PolarisSS13/Polaris/wiki/Overmap-Ships-and-Places---What's-Needed

Reply:
It's a very rough overview of it, and I've not yet written a ~proper~ guide to it.